workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/HyscaleDeployServiceCommand.java

```diff
@@ -182,13 +177,8 @@ public Integer call() throws Exception {
             logWorkflowInfo(workflowContext);
             isCommandFailed = isCommandFailed ? isCommandFailed : isServiceFailed;
         }
-        
         if (!serviceProfileMap.isEmpty()) {
```

Comment:
Since it is a print message we better not merge it in the profile. Think about this once